code/resources/css/style.css

```diff
@@ -0,0 +1,372 @@
+    box-shadow: 2px 2px 6px 0px rgba(0, 0, 0, 0.3);
+    transition: .5s ease;
+  }
+
+  .grid > article:hover {
+    box-shadow: 0 0 0 10px #B7448E;
+    transition: ease;
+  }
+  
+  .grid > article img {
+  max-width: 100%;
+  }
+  
+  .grid .text {
+  padding: 5%;
+  margin: 1%;
+  }
+  
+  .featured-image {
+  max-width: 100%;
+  height: auto;
+  }
+  
+  article {
+  padding: 8% 6%;
+  } 
+  
+  /* ----------------------------------------------- */
+  /*                  FOOTER                         */
+  /* ----------------------------------------------- */
+  
+  .footer-container {
+  background-color: #5a5858;
+  color: rgb(184, 178, 178);
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  align-items: center;
+  padding: 80px 30px;
+  }
+  
+  .footer-list {
+  list-style: none; 
+  }
+  
+  .footer-list-item {
+  margin: 20px;
+  text-align: end;
+  }
+  
+  .social-icon {
+  font-size: 40px;
+  margin: 0 6px;
+  color: rgb(184, 178, 178);
+  }
+  
+  .author-email {
+  color: rgb(184, 178, 178);
+  }
+
+
+/* ----------------------------------------------- */
+/*                  MEDIA QUERIES                  */
+/* ----------------------------------------------- */
+
+/* Big Tablets - iPad Pro */
+
+@media only screen and (max-width: 1024px) {
+
+body {
+  font-size: 16px;
+}
+
+h2, .header-subtitle {
+  font-size: 18px;
+}
+
+h3 {
+  font-size: 17px;
+}
+
+.cta-button {
+  font-size: 16px;
+  border-radius: 200px;
+  padding: 5px;
+  margin-top: 10px;
+}
+
+.grid {
+  grid-template-columns: repeat(2, 1fr);  
+} 
+
+.main-news-container {
+  flex-wrap: wrap;
+  justify-content: center;
+}
+
+.main-news-contents {
+  margin-left: 0;
+  padding: 3% 9%;
+  } 
+
+  .read-more-link {
+    font-size: 14px;
+  }
+
+}
+
+/* Small Tablets */
+
+@media only screen and (max-width: 768px) {
+
+  h2, h3 {
+    margin-top: 3%;
+  }
+
+  .featured-title {
+    margin-bottom: 10px;
+  }
+
+  .nav-bar {
+    display: none;
+  }
+
+  .hero-container {
+    background-image: -webkit-linear-gradient(rgba(0, 0, 0, 0.7), rgba(0, 0, 0, 0.7)), url(img/zbynek-burival-1kGvoC9g2DY-unsplash.jpg);
+  }
+ 
+  .main-news-contents {
+    margin-left: 0;
+    padding: 0;
+    } 
+
+  .grid {
+    grid-template-columns: repeat(2, 1fr);  
+  } 
+  
+   .footer-list {
+    text-align: center;
+  }
+ 
+  .footer-container {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-around;
+    }
+
+  .footer-list-item {
+    text-align: start;
+  }
+
+  .logo-transparent {
+    height: 100px;
+  }
+
+  .social-icon {
+    font-size: 20px;
+  }
+
+}
+
+/* Mobile - Big */
+
+@media only screen and (max-width: 767px) {
+  .grid {
+    grid-template-columns: repeat(1, 1fr);  
+  } 
+}
+
+/* Mobile - Small */
+/*For iPhone6/7/8 Plus*/ 
+
+/* @media only screen and (max-width: 414px ) {  */
+  @media only screen and (max-width: 575px ) { 
+
+  .hero-container {
+    height: 70vh;
+  }
+
+  .footer-list {
+   text-align: start;
+ }
+}
```

Comment:
I think you have done a fantastic job Isabel. It was a pleasure to read through your code and your site looks awesome. I have a lot to learn from you about writing clean code and i love that you worked a lot with % to get the optimal responsivness and less code needed to adapt to the different media queries. You should be proud of yourself!!

Reply:
Aw thank you so much Sabrina! You're great at giving feedback, I learned a lot! Your comments also encouraged me to analyze my choices and really make sure they were logical and 100% needed. I really appreciate that you took the time to check the code in depth!